### ISSUE TITLE
Immediately hangs on Windows

### ISSUE THREAD

**Issue by matthew-jones-uk** [iceiix/stevenarella#319]
Hi,

I downloaded the precompiled build from Windows and tried running. It starts downloading all the assets (and I think finishes) but then just hangs, with the game window unable to be interacted with.  The console outputs:
```
[main.rs:216][INFO] Starting steven
[blocking/wait.rs:46][TRACE] (ThreadId(2)) park without timeout
[poll.rs:787][TRACE] registering with poller
[blocking/client.rs:653][TRACE] (ThreadId(3)) start runtime::block_on
[sys/windows/selector.rs:73][TRACE] select; timeout=None
[sys/windows/selector.rs:78][TRACE] polling IOCP
[blocking/wait.rs:46][TRACE] (ThreadId(5)) park without timeout
[poll.rs:787][TRACE] registering with poller
[blocking/client.rs:653][TRACE] (ThreadId(6)) start runtime::block_on
[sys/windows/selector.rs:73][TRACE] select; timeout=None
[sys/windows/selector.rs:78][TRACE] polling IOCP
[blocking/client.rs:600][TRACE] closing runtime thread (ThreadId(3))
[blocking/client.rs:602][TRACE] signaled close for runtime thread (ThreadId(3))
[sys/windows/selector.rs:103][TRACE] returning
[blocking/client.rs:650][TRACE] (ThreadId(3)) Receiver is shutdown
[blocking/client.rs:655][TRACE] (ThreadId(3)) end runtime::block_on
[blocking/client.rs:657][TRACE] (ThreadId(3)) finished
[blocking/client.rs:604][TRACE] closed runtime thread (ThreadId(3))
```

**Comment by SuperSandro2000** [comment]
Dupe of #282

**Comment by iceiix** [comment]
Should be fixed by #282 in build 1.0.1105+ https://ci.appveyor.com/project/iceiix/stevenarella/builds/33639814
